62_CAD/common.hlsl

```diff
@@ -315,6 +334,16 @@ struct PSInput
     
     void setLineStart(float2 lineStart) { data2.xy = lineStart; }
     void setLineEnd(float2 lineEnd) { data2.zw = lineEnd; }
+
+    // Texture glyph UVs
+    // data2    
+    float2 getFontGlyphUv() { return interp_data5.xy; }
+    uint32_t getFontGlyphTextureId() { return asuint(data2.x); }
+    float2 getFontGlyphScreenSpaceSize() { return data2.yz; }
```

Comment:
from https://github.com/Devsh-Graphics-Programming/Nabla-Examples-and-Tests/pull/119/files#r1667383805
you can compute the screenPxRange and then pass it here instead of the screenSpaceSize

Reply:
Done